src/webots/gui/WbGuiApplication.cpp

```diff
@@ -480,7 +480,7 @@ bool WbGuiApplication::setup() {
   WbWrenOpenGlContext::doneWren();
 
   if (showGuidedTour)
-    mMainWindow->showGuidedTour();
+    mMainWindow->showUpdatedDialog();
```

Comment:
We should add a comment saying that the guided tour will be shown after the updated dialog.